ev3dev/brickpi.py

```diff
@@ -33,11 +33,13 @@
 OUTPUT_B = 'ttyAMA0:outB'
 OUTPUT_C = 'ttyAMA0:outC'
 OUTPUT_D = 'ttyAMA0:outD'
+OUTPUTS = (OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D)
```

Comment:
I'd rather take it out of core if it's designed to be used as a "helper". You'd get an error if you tried to open an incorrect port anyways,